### PR TITLE
[Bug](Function) fix json contains with empty value

### PR DESCRIPTION
**Pull request by eldenmoon**
## Proposed changes
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /home/zcp/repo_center/doris_release/doris/be/src/common/signal_handler.h:421
 1# os::Linux::chained_handler(int, siginfo*, void*) in /root/jdk8/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /root/jdk8/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /root/jdk8/jre/lib/amd64/server/libjvm.so
 4# 0x00007F8327EF6400 in /lib64/libc.so.6
 5# doris::JsonbValue::findValue(doris::JsonbPath&, int (*)(char const*, unsigned int)) at /home/zcp/repo_center/doris_release/doris/be/src/util/jsonb_document.h:1410
 6# doris::vectorized::JsonbContainsUtil::jsonb_contains_execute(doris::FunctionContext*, doris::vectorized::Block&, std::vector<unsigned long, std::allocator<unsigned long> > const&, unsigned lon
g, unsigned long) at /home/zcp/repo_center/doris_release/doris/be/src/vec/functions/function_jsonb.cpp:1373
 7# doris::vectorized::FunctionJsonbContains<doris::vectorized::JsonbContainsAndPathImpl>::execute_impl(doris::FunctionContext*, doris::vectorized::Block&, std::vector<unsigned long, std::allocato
r<unsigned long> > const&, unsigned long, unsigned long) const at /home/zcp/repo_center/doris_release/doris/be/src/vec/functions/function_jsonb.cpp:1309
 8# doris::vectorized::DefaultExecutable::execute_impl(doris::FunctionContext*, doris::vectorized::Block&, std::vector<unsigned long, std::allocator<unsigned long> > const&, unsigned long, unsigne
d long) const at /home/zcp/repo_center/doris_release/doris/be/src/vec/functions/function.h:429
 9# doris::vectorized::PreparedFunctionImpl::default_implementation_for_constant_arguments(doris::FunctionContext*, doris::vectorized::Block&, std::vector<unsigned long, std::allocator<unsigned lo
ng> > const&, unsigned long, unsigned long, bool, bool*) const at /home/zcp/repo_center/doris_release/doris/be/src/vec/functions/function.cpp:170
10# doris::vectorized::PreparedFunctionImpl::execute_without_low_cardinality_columns(doris::FunctionContext*, doris::vectorized::Block&, std::vector<unsigned long, std::allocator<unsigned long> >
const&, unsigned long, unsigned long, bool) const in /root/apache-doris-2.1.3-bin-x64/be/lib/doris_be
11# doris::vectorized::PreparedFunctionImpl::execute(doris::FunctionContext*, doris::vectorized::Block&, std::vector<unsigned long, std::allocator<unsigned long> > const&, unsigned long, unsigned
long, bool) const at /home/zcp/repo_center/doris_release/doris/be/src/vec/functions/function.cpp:250
12# doris::vectorized::IFunctionBase::execute(doris::FunctionContext*, doris::vectorized::Block&, std::vector<unsigned long, std::allocator<unsigned long> > const&, unsigned long, unsigned long, b
ool) const at 
Issue Number: close #xxx

### CODE REVIEW

**Comment by eldenmoon** [comment]
run buildall

**Comment by eldenmoon** [comment]
run buildall

File: regression-test/suites/query_p0/sql_functions/json_functions/test_json_function.groovy [hunk 1]
```diff
@@ -74,5 +74,7 @@ suite("test_json_function", "arrow_flight_sql") {
     qt_sql "SELECT JSON_CONTAINS('{\"name\": \"John\", \"age\": 30, \"city\": \"New York\", \"hobbies\": [\"reading\", \"travelling\"]}', '{\"age\": 31, \"hobbies\": [\"reading\"]}', '\$');"
     qt_sql "SELECT JSON_CONTAINS('{\"name\": \"John\", \"age\": 30, \"projects\": [{\"name\": \"Project A\", \"year\": 2020}, {\"name\": \"Project B\", \"year\": 2021}]}', '{\"projects\": [{\"name\": \"Project A\"}]}', '\$');"
     qt_sql "SELECT JSON_CONTAINS('{\"name\": \"John\", \"age\": 30, \"address\": {\"city\": \"New York\", \"country\": \"USA\"}}', '{\"address\": {\"city\": \"New York\"}}', '\$');"
-
+    qt_sql """SELECT JSON_CONTAINS('','1','\$.a')"""
+    qt_sql """SELECT JSON_CONTAINS('""','1','\$.a')"""
+    qt_sql """SELECT JSON_CONTAINS("",'1','\$.a')"""
```
Comment (superdiaodiao) [inline regression-test/suites/query_p0/sql_functions/json_functions/test_json_function.groovy:79]: Hi, what about adding the test for NULL by the way?

**Review by xiaokang — APPROVED** [review]
LGTM

**Review by qidaye — APPROVED** [review]
LGTM
